backend/protocol_rpc/endpoints.py

```diff
@@ -623,3 +629,7 @@ def register_all_rpc_endpoints(
         partial(get_transactions_for_address, transactions_processor, accounts_manager),
         method_name="sim_getTransactionsForAddress",
     )
+    register_rpc_endpoint(
+        partial(set_transaction_appeal, transactions_processor),
+        method_name="sim_setTransactionAppeal",
```

Comment:
```suggestion
        method_name="sim_appealTransaction",
```

Reply:
Solved in eb5ea52a2d4433e9b5e2441d05bf6f027712d486